src/QMCWaveFunctions/Fermion/MatrixDelayedUpdateCUDA.h

```diff
@@ -98,41 +75,50 @@ struct MatrixDelayedUpdateCUDAMultiWalkerMem : public Resource
 template<typename T, typename T_FP>
 class MatrixDelayedUpdateCUDA
 {
+public:
+  using WFT = WaveFunctionTypes<T, T_FP>;
```

Comment:
I think this is temporary and will go away as we fix WaveFunctionTypes propagation.